samples/bluetooth/hci_lpuart/boards/thingy91_nrf52840.conf

```diff
@@ -0,0 +1,24 @@
+# Override prj.conf defaults
+
+CONFIG_GPIO=y
+CONFIG_BT_CTLR=y
+CONFIG_MAIN_STACK_SIZE=1024
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=512
+CONFIG_BT_WAIT_NOP=y
+CONFIG_BT_MAX_CONN=16
+CONFIG_BT_TINYCRYPT_ECC=n
+CONFIG_BT_CTLR_DTM_HCI=y
+CONFIG_BT_CTLR_ASSERT_HANDLER=y
+
+CONFIG_UART_1_ASYNC=y
+CONFIG_UART_1_INTERRUPT_DRIVEN=n
+CONFIG_UART_1_NRF_HW_ASYNC=y
+CONFIG_UART_1_NRF_HW_ASYNC_TIMER=2
+CONFIG_NRF_SW_LPUART=y
+CONFIG_NRF_SW_LPUART_INT_DRIVEN=y
+
+CONFIG_GPIO_AS_PINRESET=y
+
+CONFIG_USE_SEGGER_RTT=y
+CONFIG_RTT_CONSOLE=y
+CONFIG_UART_CONSOLE=n
```

Comment:
Why is this needed ?
Seems to be `n` per default.

Reply:
Inherited from the nRF9160DK configuration, but I've removed it.

Reply:
Correct. Fixed.